src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -39,4 +39,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
Remember to update your method with header comments as your method has a different behaviour from other  subclass implementations. e.g. Returns if true if class mutates data.